moped-editor/src/views/projects/projectView/ProjectComponents/TheMap.js

```diff
@@ -105,7 +105,7 @@ export default function TheMap({
         type: "show_error",
         payload: { message: "Zoom in to select features", severity: "error" },
       });
-    } else if (isCreatingComponent || isEditingComponent) {
+    } else {
```

Comment:
This fixes a bug where the **Zoom in to select features** warning that appear when mapping component features would stick around if you started creating or editing and then pressed the **Cancel** button. It nows disappears when cancelling out.